README.md

```diff
@@ -362,6 +363,27 @@ MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py build --cmake
 ccmake build  # or cmake-gui build
 ```
 
+##### Installation Issue
+
+Sometimes you would encounter PyTorch local source is not being detected by Python interpreter after building PyTorch above.
+To resolve this, you may try the following steps:
+
+1. Remove the previously installed package:
+
+    ```bash
+    python setup.py develop --uninstall
```

Comment:
can also just pip uninstall torch

Reply:
I don't think this will remove `.egg` file, at least not with the oldest versions of pip. I.e. one can `pip install torch; python setup.py develop` and in that case `pip uninstall torch` will remove the wheel, rather than egg